infra/base-images/base-builder/compile_afl

```diff
@@ -51,7 +51,10 @@ export ASAN_OPTIONS="detect_leaks=0:symbolize=0:detect_odr_violation=0:abort_on_
 rm -f "$OUT/afl_cmplog.txt"
 test $(($RANDOM % 10)) -lt 4 && {
   export AFL_LLVM_CMPLOG=1
-  # We need to notify afl-fuzz to activate CMPLOG
+}
+
+# We need to notify afl-fuzz to activate CMPLOG
+test -n "$AFL_LLVM_CMPLOG" && {
```

Comment:
This is causing 

/usr/local/bin/compile_afl: line 57: AFL_LLVM_CMPLOG: unbound variable
Building fuzzers failed.

need to define it if it does not run into that branch.